evap/staff/importers.py

```diff
@@ -369,15 +392,43 @@ def save_users_to_db(self):
                     new_participants.append(user)
                     if created:
                         users_count += 1
+                        created_users.append(user)
 
                 except Exception as e:
                     self.errors.append(_("A problem occured while writing the entries to the database."
                                          " The original data location was row %(row)d of sheet '%(sheet)s'."
                                          " The error message has been: '%(error)s'") % dict(row=row+1, sheet=sheet, error=e))
                     raise
-        self.success_messages.append(_("Successfully created %(users)d user(s).") % dict(users=users_count))
+
+        msg = _("Successfully created {} user(s):").format(users_count)
+        for user in created_users:
+            msg += "<br>"
+            msg += ("{} {} ({})").format(user.first_name, user.last_name, user.username)
+        self.success_messages.append(mark_safe(msg))
         return new_participants
 
+    def get_user_profile_list(self):
+        new_participants = []
+        for user_data in self.users.values():
+            if UserProfile.objects.filter(username=user_data.username).exists():
+                new_participants.append(UserProfile.objects.get(username=user_data.username))
```

Comment:
to avoid the two queries:
```
try:
    new_participant = UserProfile.objects.get(username=user_data.username)
except: 
    new_participant = UserProfile(...)
new_participants.append(new_participant)
```